@uportal/esco-content-menu/README.md

```diff
@@ -242,6 +242,32 @@ This component render informations about a portlet as a card.
 - `favorite-api-url`: type: `String`, default: `/uPortal/api/layout`, the uri/url of the favorites api (passed to the `action-favorite` component),
 - `user-info-api-url`: type: `String`, default: `/uPortal/api/v5-1/userinfo`, url/uri on which the api request is done to obtain user information and the jwt tocken, (passed to the `action-favorite` component),
 
+### The content grid category filter
+
+This component is an external version of the filter built into Content Grid that allows for arbitrary placement on the page.
+
+#### Html
+
+```html
+<script src="/resource-server/webjars/vue/dist/vue.min.js"></script>
+<script
+  src="/resource-server/webjars/uportal__esco-content-menu/dist/esco.min.js"
+  defer
+></script>
+
+<esco-content-grid
+  portlet-card-size="gridPortletCardSize"
+  hide-action="hideAction"
+>
+</esco-content-grid>
+
+<esco-content-grid-filter />
```

Comment:
The HTML spec restricts custom elements to non-self-closing only.

```suggestion
<esco-content-grid-filter></esco-content-grid-filter>
```

Reply:
`npm` complains about it

Reply:
```
Module Warning (from ./node_modules/eslint-loader/index.js):
error: Require self-closing on Vue.js custom components (<content-grid-filter>) (vue/html-self-closing) at src/App.vue:3:5:
  1 | <template>
  2 |   <div id="app">
> 3 |     <content-grid-filter></content-grid-filter>
    |     ^
  4 |     <content-grid
  5 |       background-color="white"
  6 |       show-footer-categories


1 error found.
1 error potentially fixable with the `--fix` option.

```

Reply:
This is the README. app.js should not be impacted.